src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+          }
+        }
+      }
+    }
+  }
+```
+
+Here, each environment configuration has the following parameters:
+
+```graphql
+"Environment configuration object"
+type EnvironmentConfiguration {
+  "Flapping interval in seconds"
+  flappingInterval: Seconds
+  "Grace periods per priority"
+  gracePeriod: [GracePeriodConfig!]
+  "Inactive period in seconds"
+  inactivePeriod: Seconds
+  "Incident timeout in seconds"
+  incidentTimeout: Seconds
+  "Issue TTL in seconds"
+  issueTtl: Seconds
+  "Max incidents in the issue"
+  maxIssueSize: Int
+}
+```
+
+Using the corresponding `mutations` you can also change `issueTt`l and `gracePeriod` environment configuration parameters.
+
+* aiIssuesUpdateIssueTtl
+* aiIssuesUpdateGracePeriod
+
+**Request**
+
+```graphql
+mutation {
+  aiIssuesUpdateIssueTtl(accountId: YOUR_ACCOUNT_ID, issueTtl: 259200) {
+    error
+    config {
+      flappingInterval
+      gracePeriod {
+        period
+        priority
+      }
+      inactivePeriod
+      incidentTimeout
+      issueTtl
+      maxIssueSize
+    }
+  }
+}
+```
+
+**Results**
+
+```json
+{
+  "data": {
+    "aiIssuesUpdateIssueTtl": {
+      "config": {
+        "flappingInterval": 0,
+        "gracePeriod": [
+          {
+            "period": 60,
+            "priority": "CRITICAL"
+          },
+          {
+            "period": 300,
+            "priority": "HIGH"
+          },
+          {
+            "period": 600,
+            "priority": "MEDIUM"
+          },
+          {
+            "period": 900,
+            "priority": "LOW"
+          }
+        ],
+        "inactivePeriod": 14400,
+        "incidentTimeout": 90000,
+        "issueTtl": 259200,
+        "maxIssueSize": 5000
+      },
+      "error": null
+    }
+  },
+
+```
+ </Collapser>
+</CollapserGroup>
+
+
+## Acknowledge or unacknowledge issues [#resolve-ack-issues]
+
+Using `mutations` you can acknowledge or unacknowledge and resolve the given issue by its   issueId . 
+
+* aiIssuesAckIssue
+* aiIssuesResolveIssue
+* aiIssuesUnackIssue
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="ackissues"
+    title="Acknowledge or unacknowledge issues"
+  >
+
+**Request**
+
+```graphql
+mutation {
+  aiIssuesResolveIssue(accountId: 10581561, issueId: "f0846f5e-fc9d-4dc0-afdd-07cce7cc6fc4") {
+    error
+    result {
+      action
+      accountId
+      issueId
+      routingKey
+    }
+  }
+}
+```
+
+**Results**
+
+```json
+{
+"data": {
+    "aiIssuesResolveIssue": {
+      "error": null,
+      "result": {
+        "accountId": 10581561,
+        "action": "RESOLVE",
+        "issueId": "f0846f5e-fc9d-4dc0-afdd-07cce7cc6fc4",
+        "routingKey": "93538a443af520d10e7999fdb183ebf21e4cb6bd"
+      }
+    }
+```
+ </Collapser>
+</CollapserGroup>
+
+
+## Running from the command line [#running-from-command-line]
+
+To run queries from the command line:
+
+1 Get your API key, fetch or create one in the [API Key UI](https://staging-one.newrelic.com/api-keys-ui).
```

Comment:
we should specify this is the 'user key' they need. also note the bad list markdown.